megamek/src/megamek/common/PlanetaryConditions.java

```diff
@@ -650,9 +650,19 @@ public int getVisualRange(Entity en, boolean targetIlluminated) {
             isAero = (en.isAero()) && !isLargeCraft;
         }
         //anything else is infantry
-
-        if ((lightConditions == L_PITCH_BLACK) && !Spotlight
-                && !targetIlluminated) {
+        
+        //New rulings per v3.02 errata. Spotlights are easier, yay!
+        //Illuminated?  Flat 45 hex distance
+        if (targetIlluminated && lightConditions > L_DAY) {
+            return 45;
+        } else if (Spotlight && !targetIlluminated && lightConditions > L_DAY) {
```

Comment:
```suggestion
        } else if (Spotlight && (lightConditions > L_DAY)) {
```

Reply:
!targetIlluminated is always true because of the previous check

Reply:
Point. I was thinking to catch the edge case where lighting IS day, but it didn't do that either.